src/components/AuthStoreSetter.ts

```diff
@@ -19,6 +20,9 @@ function AuthStoreSetter(): null {
       auth.events.addUserLoaded((user: OidcUser) => {
         loadOidcUser(user);
       }),
+      auth.events.addAccessTokenExpiring((): void => {
```

Comment:
I could have sworn that we used to have a handler for this event.

We configured the expiring notification time in `getOidcConfig`.

Reply:
I believe the time configured in`getOidcConfig` is not being used in the normal auth flow (as per the comment this is related t0 Leo's setCookie interval).

But you just made me realize something. Through testing, this event is fired 55 minutes after the authToken expires. So this setting may be the time before the authToken expires which is sends this event. So for testing, we can set this to be "expiring" after 1 minute.

Reply:
I had assumed that `accessTokenExpiringNotificationTimeInSeconds` was the time after the auth token has been given ad this just was not working, but it is counting from the other direction, it appears.

Reply:
> I believe the time configured in `getOidcConfig` is not being used in the normal auth flow (as per the comment this is related t0 Leo's setCookie interval)

It's configuration for `UserManager`. I believe the intent was to use that notification to sync the Leo cookie updates (the cookie contains a copy of the auth token) with the auth token refresh. (#3517)